File: chinese-shadowing-api/Shadowing.DataAccess/Entities/Shadow.cs
using System;

namespace Shadowing.DataAccess.Entities
{
    public class Shadow
    {
        public string Id { get; set; }
        public string SectionId { get; set; }
        public Section Section { get; set; }
        public string RecordedUrl { get; set; }
        public string InferenceText { get; set; }
        public string RoomId { get; set; }
        public Room Room { get; set; }
        public DateTime Created { get; set; }
        public DateTime? Updated { get; set; }
    }
}

File: chinese-shadowing-client/src/models/Shadow.js
export default class Shadow {
  /** @type {string} */
  id;
  /** @type {string} */
  sectionId;
  /** @type {string} */
  recordedUrl;
  /** @type {string} */
  inferenceText;

  constructor(shadow) {
    this.id = shadow.id;
    this.sectionId = shadow.sectionId;
    this.recordedUrl = shadow.recordedUrl;
    this.inferenceText = shadow.inferenceText;
  }
}
